devops/ansible/roles/rpc/tasks/main.yml

```diff
@@ -48,7 +48,6 @@
   environment:
     JOYSTREAM_NODE_WS: 'ws://{{ inventory_hostname }}:9944/'
     # TREASURY_ACCOUNT_URI: '{% if endowed_key is defined %}{{ endowed_key }}{% else %}{{ sudo_key }}{% endif %}'
```

Comment:
Will drop all env variables here expect the ws endpoint.
The other variables where there because previously instead of starting only query-node the playbook also attempted to start a faucet, and initialized it with the faucet setup scenario from integration tests.